webrick/upstart/tasks/RedHat.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: systemd unit - RedHat
+  template:
+    dest: "{{ webrick_systemd_unit }}"
+    src: systemd.service.j2
```

Comment:
Yes, I had intended to add a comment about that. I would prefer to just rename the role. *service* would match ansible's name for the generalized init-system module. But it is also agreeable for me to put it in a seperate role.
 
It doesn't seem appropriate to me to run the the Rails-server as a *userjob*. 
  1. As far as I understand it, the purpose of systemd's user-units is only to be run (or automatically started) when (while) the user logs in (is logged in). 
  1. The Rails-server is a constitutive part of the server as a whole like the webserver, the db-server a.s.o.

And it runs as the `deploy`-user with limited permissions. 

But maybe I just don't understand the real benefit of allowing the service to be restarted without root rights?